tests/expected/lsfd/mkfds-tcp6

```diff
@@ -0,0 +1,4 @@
+    3 TCPv6   SOCK state=listen laddr=[::1]:34567                             listen    stream              1         ::1          :: ::1:34567     34567      :::0         0
+    4 TCPv6   SOCK state=established laddr=[::1]:23456 raddr=[::1]:34567 established    stream              0         ::1         ::1 ::1:23456     23456 ::1:34567     34567
+    5 TCPv6   SOCK state=established laddr=[::1]:34567 raddr=[::1]:23456 established    stream              0         ::1         ::1 ::1:34567     34567 ::1:23456     23456
```

Comment:
Shouldn't the later host:port combos also be `[::1]:34567`?

Reply:
Thank you. You are correct. I'll fix this.

Reply:
Updated.